examples/model_selection/plot_roc.py

```diff
@@ -215,7 +211,13 @@
 #
 # Obtaining the macro-average requires computing the metric independently for
 # each class and then taking the average over them, hence treating all classes
-# equally a priori. We first aggregate the true/false positive rates per class:
+# equally a priority. We first aggregate the true/false positive rates per class:
```

Comment:
```suggestion
# equally a priori. We first aggregate the true/false positive rates per class:
```
I believe 'a priori' (from Latin) is correctly used here.

Reply:
@yuanx749 thanks!